core/management/commands/queue_warm_cache.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from chronam.core.models import Page
+from chronam.core import tasks
+
+_logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = "Queue a celery request for each page from chronicling america to warm up the cache."
+
+    def handle(self, **options):
+        for page in Page.objects.all():
```

Comment:
Won't this be somewhat expensive queuing millions of pages (which may not even all fit in the cache)? Should this command accept something like a batch name or a time value to e.g. select pages modified in the last `n` minutes?

Reply:
It'd depend on the task queue overhead but if this will operate on very large lists I generally find better performance by chunking the operations so you don't take the full startup / teardown cost on every request. For example, that can be using something like [`more_itertools`’s `chunked` iterator](https://more-itertools.readthedocs.io/en/latest/api.html#more_itertools.chunked) to queue a few dozen URLs at once.